hushline/model/field_value.py

```diff
@@ -36,13 +36,17 @@ def add_padding(value: str, block_size: int = 512) -> str:
 class FieldValue(Model):
     __tablename__ = "field_values"
 
-    id: Mapped[int] = mapped_column(primary_key=True)
-    field_definition_id: Mapped[int] = mapped_column(db.ForeignKey("field_definitions.id"))
+    id: Mapped[int] = mapped_column(primary_key=True, nullable=False)
+    field_definition_id: Mapped[int] = mapped_column(
+        db.ForeignKey("field_definitions.id", ondelete="CASCADE"), nullable=True
```

Comment:
this cascade shouldn't be needed because nothing depends on a `FieldValue`, only the other way around

Reply:
If I remove the cascade here, it tries to autogenerate this migration:

```py
def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('field_values', schema=None) as batch_op:
        batch_op.drop_constraint('fk_field_values_field_definition_id_field_definitions', type_='foreignkey')
        batch_op.create_foreign_key(batch_op.f('fk_field_values_field_definition_id_field_definitions'), 'field_definitions', ['field_definition_id'], ['id'])

    # ### end Alembic commands ###
```

So I think this is something else we should keep and fix in a future PR that includes an actual migration.